dask/base.py

```diff
@@ -1169,6 +1179,27 @@ def _normalize_function(func: Callable) -> tuple | str | bytes:
             kws = None
         return (normalize_function(func.func), args, kws)
     else:
+        try:
+            co = func.__code__
+            # https://docs.python.org/3/library/inspect.html#types-and-members
+            return (
+                co.co_argcount,
+                co.co_code,
+                co.co_cellvars,
+                co.co_consts,
+                # co.co_filename,
+                # co.co_firstlineno,
+                # co.co_flags,
+                # co.co_lnotab,
```

Comment:
I modeled this after an issue that was opened a couple of months ago. I don't have a strong prefernence whether line numbers should count here. See test `test_tokenize_local_functions`

This somewhat overlaps with https://github.com/dask-contrib/dask-expr/pull/689 cc @milesgranger